C/plugins/north/OMF/linkdata.cpp

```diff
@@ -502,3 +555,69 @@ bool OMFLinkedData::flushContainers(HttpSender& sender, const string& path, vect
 	}
 	return true;
 }
+
+/**
+ * Set the base tyoe by passign the string of the base type
+ */
+void LALookup::setBaseType(const string& baseType)
+{
+	if (baseType.compare("Double64") == 0)
+		m_baseType = OMFBT_DOUBLE64;
+	else if (baseType.compare("Double32") == 0)
+		m_baseType = OMFBT_DOUBLE32;
+	else if (baseType.compare("Integer16") == 0)
+		m_baseType = OMFBT_INTEGER16;
+	else if (baseType.compare("Integer32") == 0)
+		m_baseType = OMFBT_INTEGER32;
+	else if (baseType.compare("Integer64") == 0)
+		m_baseType = OMFBT_INTEGER64;
+	else if (baseType.compare("UInteger16") == 0)
+		m_baseType = OMFBT_UINTEGER16;
+	else if (baseType.compare("UInteger32") == 0)
+		m_baseType = OMFBT_UINTEGER32;
+	else if (baseType.compare("UInteger64") == 0)
+		m_baseType = OMFBT_UINTEGER64;
+	else if (baseType.compare("String") == 0)
+		m_baseType = OMFBT_STRING;
+	else if (baseType.compare("FledgeAsset") == 0)
+		m_baseType = OMFBT_FLEDGEASSET;
```

Comment:
Add a final "_else_" that sets the type to _OMFBT_UNKNOWN_ just in case.

Reply:
I see that you now log a fatal message. Do you also need to set _m_baseType_ to _OMFBT_UNKNOWN_?

Reply:
No, it should never happen